README.md

```diff
@@ -877,31 +877,86 @@ interface FIDAttribution {
 ```ts
 interface INPAttribution {
   /**
-   * A selector identifying the element that the user interacted with for
-   * the event corresponding to INP. This element will be the `target` of the
-   * `event` dispatched.
+   * A selector identifying the element that the user first interacted with
+   * as part of the frame where the INP candidate interaction occurred.
+   * If `interactionTarget` is an empty string, that generally means the
+   * element was removed from the DOM as part of the interaction.
    */
-  eventTarget?: string;
+  interactionTarget: string;
   /**
-   * The time when the user interacted for the event corresponding to INP.
-   * This time will match the `timeStamp` value of the `event` dispatched.
+   * The time when the user first interacted during the frame where the INP
+   * candidate interaction occurred (if more than one interaction occurred
+   * within the frame, only the first time is reported).
    */
-  eventTime?: number;
+  interactionTime: number;
   /**
-   * The `type` of the `event` dispatched corresponding to INP.
+   * The type of interaction. This will be either 'pointer' or 'keyboard'
+   * since those are the only types of interactions considered for INP.
    */
-  eventType?: string;
+  interactionType: 'pointer' | 'keyboard';
```

Comment:
This is to better align with what's shown in DevTools. Also, since now there's no single `eventEntry` that is reported, it doesn't make sense to expose a single event type.

Reply:
> what about cases where the second interaction in the frame was the one that contributed the most processing time, and so was really "the longest"? Could that occasionally be confusing?

I do think it could be confusing, but I think that's a general problem/challenge with the INP definition and not something specific to this library.

That said, with the changes in this PR it should be possible for a developer to determine what other events were processed in the same animation frame, so hopefully that addresses the root of your concern.